site/src/pages/components/customizableselect.mdx

```diff
@@ -245,6 +263,9 @@ This example has a complex structure wrapping the `<option>`s which is all slott
 .container > div {
   background-color: lightgray;
 }
+select, select::picker(select) {
```

Comment:
I think **all** of these should be simplified:

```
select, ::picker(select) {
```

(the `::picker(select)` can only ever match on a `<select>` element.)

Reply:
I simplified all the code examples, but I kept it as select::picker(select) in some of the explanatory text because I think it's helpful to know that it applies to the select element